js/core/devices.js

```diff
@@ -84,8 +86,10 @@ const uaParsers = {
     },
 
     android(userAgent) {
-        if(!/android|htc_|silk/i.test(userAgent)) {
-            return;
+        // TODO: Check this RegExp.
+        //  It looks like there may be missing android user agents.
+        if(!/android|htc_|silk/i.test(userAgent) || /windows phone/i.test(userAgent)) {
+            return null;
```

Comment:
can we rewrite it like this:
```suggestion
        const isAndroid = /android|htc_|silk/i.test(userAgent)
        const isWindowsPhone = /windows phone/i.test(userAgent)
        
        if(!isAndroid || isWindowsPhone) {
          return null;
        }
```

Reply:
Yes, I agree with you. 
Thank you for the great suggestion!